lux/_config/config.py

```diff
@@ -177,5 +179,50 @@ def default_display(self, type: str) -> None:
                 stacklevel=2,
             )
 
+    @property
+    def default_renderer(self):
```

Comment:
Do we actually need `default_renderer` and `default_plot_config` as additional things inside the config or should we just have this as a variable set from `renderer` and `plot_config`?